src/main/java/edu/greenblitz/tobyDetermined/Robot.java

```diff
@@ -1,30 +1,35 @@
 package edu.greenblitz.tobyDetermined;
```

Comment:
The only change needed in this file is the led instance

Reply:
we need to set it's default command